target/linux/realtek/dts-5.15/rtl9302_zyxel_xgs1250-12.dts

```diff
@@ -181,6 +283,7 @@
 			sds = < 6 >;
 			// Disabled because we do not know how to bring up again
 			// reset-gpios = <&gpio0 21 GPIO_ACTIVE_LOW>;
+			#thermal-sensor-cells = <0>;
```

Comment:
I didn't know this phy driver exposed hwmon devices. That's pretty neat :)

Reply:
Yep, it is a nice feature. Also very useful or even almost a necessity with how power hungry 10G Base-T phys are...